### PR TITLE
Tape fix

### PR DESCRIPTION
**Pull request by Thenerdstation**
Fix failing test in tape mode.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 projectq>=0.5.1
-pennylane>=0.11
+git+https://github.com/PennyLaneAI/pennylane.git
```
Comment (mariaschuld) [inline requirements.txt:2]: So this would be reverted after release, right?

**Review by mariaschuld — APPROVED** [review]
Nice and easy one, thanks!

**Comment by Thenerdstation** [comment]
@josh146 This included changes to requirements.txt just for the build so we should revert that when the next pennylane is released

**Comment by josh146** [comment]
Yep, saw that, thanks for the reminder! 👍 

I merged this in partly because (a) I saw codecov was not working, so everyone else was blocked from merging it, and (b) to allow it to be marked as green on the plugin test repo.

**Pull request merged**